plugins/com.google.cloud.tools.eclipse.appengine.localserver/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-Activator: com.google.cloud.tools.eclipse.appengine.localserver.Activator
 Bundle-ActivationPolicy: lazy
-Require-Bundle: com.google.cloud.tools.appengine;bundle-version="0.1.0",
+Require-Bundle: com.google.cloud.tools.appengine;bundle-version="0.1.14",
```

Comment:
Isn't it 0.1.13?

https://github.com/GoogleCloudPlatform/appengine-plugins-core/commits/v0.1.13